### PR TITLE
Fix for default font name in VideoPreviewFont

### PR DESCRIPTION
**Pull request by OmrSi**
Fix for the issue mentioned here https://github.com/SubtitleEdit/subtitleedit/issues/4705#issuecomment-758544771

### CODE REVIEW

File: src/libse/Common/Settings.cs [hunk 1]
```diff
@@ -1323,6 +1314,17 @@ public GeneralSettings()
             DialogStyle = DialogType.DashBothLinesWithSpace;
             ContinuationStyle = ContinuationStyle.None;
 
+            if (Configuration.IsRunningOnLinux)
+            {
+                SubtitleFontName = Configuration.DefaultLinuxFontName;
+                VideoPlayerPreviewFontName = Configuration.DefaultLinuxFontName;
+            }
+            else if (Environment.OSVersion.Version.Major < 6 || Environment.OSVersion.Version.Major == 6 && Environment.OSVersion.Version.Minor < 2) // Vista/Win2008Server/Win7
```
Comment (OmrSi) [inline src/libse/Common/Settings.cs:1322]: As you can see here https://stackoverflow.com/a/2819962/14349868
Windows 7 is 6, but the minor is 1, so it should be like this, I think.

Comment (niksedk) [reply src/libse/Common/Settings.cs:1322]: I think you can remove "else stuff"" as  WinXP is no longer supported.
"Times New Roman" was one of the few fonts on WinXP that had some Unicode support.

Comment (OmrSi) [reply src/libse/Common/Settings.cs:1322]: So Windows 7 has "Tahoma"? The comment mentions Windows 7 as well.

Side note: I moved from XP to 10, never worked with 7. ^^"

Comment (OmrSi) [reply src/libse/Common/Settings.cs:1322]: Okay, given that the code was using Tahoma on Win7 and no one complained, I guess it's safe to say that it works on Win7.

Comment (niksedk) [reply src/libse/Common/Settings.cs]: Yes, Unicode support in Win7 is fine :)

**Pull request merged**